lib/client.js

```diff
@@ -126,6 +126,7 @@ Client.prototype._initializeOptions = function(options) {
   this.streamAllowed = options.stream;
   this.wsdl.options.attributesKey = options.attributesKey || 'attributes';
   this.wsdl.options.envelopeKey = options.envelopeKey || 'soap';
+  this.wsdl.options.preserveWhitespace = options.preserveWhitespace || false;
```

Comment:
could you please use `!!` to convert into boolean (`this.wsdl.options.preserverWhitespace = !!options.preserveWhitespace`)?